salt/modules/yumpkg.py

```diff
@@ -580,8 +580,9 @@ def latest_version(*names, **kwargs):
     cmd = [_yum(), '--quiet']
     cmd.extend(repo_arg)
     cmd.extend(exclude_arg)
-    cmd.extend(['list', 'available'])
+    cmd.extend(['list'])
     cmd.extend(names)
+    log.debug(cmd)
```

Comment:
Was log statement left here intentionally? If so, might be nice to put a little more context around the message. Though, I thought the cmd module would print the command anyway with debug logging?